ekwipunek.py

```diff
@@ -0,0 +1,823 @@
+
+
+class Zywnosc(dict):
+    """
+    Kontener na przedmioty klasy Jedzenie posiadający metodę wyświetl
+    """
+    def wyswietl(self):
+        return self
+
+
+class Pozostalosci(dict):
+    """
+    Kontener na przedmioty klasy Pozostale posiadający metodę wyświetl
+    """
+    def wyswietl(self):
+        return self
+
+class Ekwipunek:
+    """
+    Klasa Ekwipunek -> To w niej znajdują się Przedmioty z Podłogi dodane za pomocą
+    Bohatera. Tutaj również znajduje się kontener w uzyciu (nie wiem, czy słusznie).
+    :param brone: kontener na przedmioty klasy Broń
+    :param pancerze: kontener na przedmioty klasy Pancerz
+    :param przedmioty magiczne: kontener na przedmioty klasy Magia
+    :param pisma: kontener na przedmioty klasy Pismo
+    :param artefakty: kontener na przedmioty klasy Artefakt
+    :param jedzenie: kontener na przedmioty klasy Jedzenie
+    :param pozostale: kontener na przedmioty klasy Pozostale
+    :param magazyn: słownik zawierający wszystkie kontenery przypisane
+        do odpowiedniego klucza
+    :param w_uzyciu: kontener zawierający przedmioty znajdujące się w danej
+        chwili w użyciu
+    """
+    def __init__(self):
+        self._bronie = Bronie()
+        self._pancerze = Pancerze()
+        self._przedmioty_magiczne = PrzedmiotyMagiczne()
+        self._pisma = Pisma()
+        self._artefakty = Artefakty()
+        self._jedzenie = Zywnosc()
+        self._pozostale = Pozostalosci()
+        self._magazyn = {
+            "Bronie": self._bronie,
+            "Pancerze": self._pancerze,
+            "Przemioty Magiczne": self._przedmioty_magiczne,
+            "Pisma": self._pisma,
+            "Artefakty": self._artefakty,
+            "Jedzenie": self._jedzenie,
+            "Pozostałe": self._pozostale,
+        }
+        self._w_uzyciu = {}
+
+    @property
+    def bronie(self):
+        return self._bronie
+
+    @property
+    def pancerze(self):
+        return self._pancerze
+
+    @property
+    def przedmioty_magiczne(self):
+        return self._przedmioty_magiczne
+
+    @property
+    def pisma(self):
+        return self._pisma
+
+    @property
+    def artefakty(self):
+        return self._artefakty
+
+    @property
+    def jedzenie(self):
+        return self._jedzenie
+
+    @property
+    def pozostale(self):
+        return self._pozostale
+
+    @property
+    def w_uzyciu(self):
+        return self._w_uzyciu
+
+    @property
+    def magazyn(self):
+        return self._magazyn
+
+    def wyrzuc(self):
+        """
+        Metoda wyrzuć wyrzucająca dany przedmiot z obiektu klasy Ekwipunek i zwracająca go
+        do obiektu klasy Podłoga.
+        """
+        print("Przedmioty, które możesz wyrzucić, lub wpisz 'wyjscie', by wyjść:")
+        for key, value in self.wyswietl_magazyn().items():
+            for k, v in value.items():
+                print(k + " sztuk " + str(len(v)))
+        val = input("Podaj nazwę przedmiotu, który chcesz wyrzucić")
```

Comment:
Nazwa przedmiotu to bardzo dużo do podania. Lepiej do każdego przedmiotu zdefiniowac id i na jego podstawie go wyrzucić ( taki odpowiednik strzałek, tam tez nic nie wpisujesz )

Reply:
Ale przecież mam nie wiedzieć, który przedmiot konkretnie wyrzucam, a jeśli wyrzucę ten z danym ID to w końcu wiem, który wyrzucam